.circleci/config.yml

```diff
@@ -37,6 +37,14 @@ commands:
   check-changed-files-or-halt:
     steps:
       - run: ./scripts/check-file-changes.sh
+  rebase-on-master:
+    steps:
+      - run: |
+          git config --global user.email "telegraf@influxdb.com"
+          git config --global user.name "telegraf"
+          git remote add upstream https://github.com/influxdata/telegraf
+          git fetch upstream
```

Comment:
I think these 2 can be combined and also limited to only fetch master branch. Now there is a lot of noise in the output of this step because it's fetching all branches